models/running_parsers.go

```diff
@@ -0,0 +1,125 @@
+package models
+
+import (
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/selfstat"
+)
+
+type RunningParser struct {
```

Comment:
As this is some larger rework, is getting rid of the runners a must?

The issue of having multiple parsers is partially mitigated by this PR as the configuration options (and toml annotations) move to the parsers instead of keeping them in a central configuration. This way you can define multiple parsers with subtables in the TOML and handle them with ease. But yeah you have to change (or at least extend) the TOML structure for this... There might be a way to do this in a backward compatible way but that's not for this PR I think.